### DISCUSSION TITLE
Questions regarding UI

### DISCUSSION THREAD

**Discussion by verisgit** [basking-in-the-sun2000/solar-logger#56] [Q&A]
Hi there,

Been running this successfully for some time now but have been meaning to ask a few questions regarding the UI.

What does the purple Max represent?
![image](https://user-images.githubusercontent.com/18733413/201510167-e86e51ff-6ea0-4926-9585-84cdc1076086.png)

Why are there gaps in the collection data? it always fluctuates between a true reading and 0 so the charts all have gaps in them. Is there a way to resolve that or is it just how tit is?

![image](https://user-images.githubusercontent.com/18733413/201510208-8d9d8b0a-283a-40ad-b4d1-3c08978d55e9.png)

I never managed to get these four widgets to work. Not sure what they are:
![image](https://user-images.githubusercontent.com/18733413/201510216-fcff51df-c07c-41f7-ab2d-e442c112bd7f.png)

What would you recommend is the best way to customise this? 
![image](https://user-images.githubusercontent.com/18733413/201510232-cf4f29e5-bf48-45a4-83a2-0007e211ea53.png)

What does this forecast represent? 
![image](https://user-images.githubusercontent.com/18733413/201510264-c3e116f2-6b66-4e56-84f8-ec424e880f5c.png)

Thanks for the help.

**Comment by basking-in-the-sun2000** [comment]
> What does the purple Max represent? ![image](https://user-images.githubusercontent.com/18733413/201510167-e86e51ff-6ea0-4926-9585-84cdc1076086.png)
The purple (max) is the maximum value the inverter produced, while the green (Inverter) is the average power produced.  The lilac (limit) is the value of the maximum power the inverter can produce. The defaults are probably higher than what your system is producing, even though it seems to reach a maximum value. My inverter doesn't do a peak power, but current. The limit is the product of the average voltage and the current. Understand some newer inverters will adjust the max current to always limit the power to their nameplate value


> Why are there gaps in the collection data? it always fluctuates between a true reading and 0 so the charts all have gaps in them. Is there a way to resolve that or is it just how tit is?
The logger didn't get that point (30s). Either you had a network issue, or the the cpu is not keeping up with the influx

> 
> ![image](https://user-images.githubusercontent.com/18733413/201510208-8d9d8b0a-283a-40ad-b4d1-3c08978d55e9.png)
> 
> I never managed to get these four widgets to work. Not sure what they are:
If you just started these values don't have enough data. You need a month for the earliest one to show up a value
 ![image](https://user-images.githubusercontent.com/18733413/201510216-fcff51df-c07c-41f7-ab2d-e442c112bd7f.png)
> 
> What would you recommend is the best way to customise this? 
Not sure what you mean. You need to adjust the threshold values for the colors, and some of the limits to reflect your site. 
![image](https://user-images.githubusercontent.com/18733413/201510232-cf4f29e5-bf48-45a4-83a2-0007e211ea53.png)
> 
> What does this forecast represent?
If you are using solcast, it is the expected daily balance. It is showing that you will have a 4.5 kWh excess. Been trying to fine tune it, since it  fluctuates a bit. 
 ![image](https://user-images.githubusercontent.com/18733413/201510264-c3e116f2-6b66-4e56-84f8-ec424e880f5c.png)

**Comment by verisgit** [comment]
![image](https://user-images.githubusercontent.com/18733413/205217470-cd0377ba-3189-4e34-80a6-6aa15a9f3f01.png)
It's been running for several months now.

**Reply by basking-in-the-sun2000** [reply]
did you create the continuous query (check the end of the readme file)? I started compacting the data after a few months using a continuous query, since influxdb had issues because of the size. Glad you aren't ;)

Unless you have a special need, you probably won't need the 30s data after a few months.
